android/build.gradle

```diff
@@ -44,6 +44,15 @@ android {
     defaultConfig {
         minSdkVersion 21
     }
+
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+        kotlinOptions {
+            freeCompilerArgs += [
+                    '-Xopt-in=kotlin.RequiresOptIn',
+                    '-Xopt-in=io.customer.base.internal.InternalCustomerIOApi',
+            ]
+        }
+    }
```

Comment:
Could you leave a comment saying why this is added? Help future code readers understand the decision behind adding this.